src/registerProviders.ts

```diff
@@ -8,34 +8,40 @@ import { LocalFileSystemProxy } from "./providers/storage/localFileSystemProxy";
 import { StorageProviderFactory } from "./providers/storage/storageProviderFactory";
 import registerToolbar from "./registerToolbar";
 import { strings } from "./common/strings";
+import { HostProcessType } from "./common/hostProcess";
 
 export default function registerProviders() {
     // Storage Providers
     StorageProviderFactory.register({
         name: "localFileSystemProxy",
         displayName: strings.connections.providers.local.title,
+        platformSupport: HostProcessType.All,
```

Comment:
Let's default the platform support for all within the registration method... if not defined in the options we will assume it is all unless otherwise specified.